packages/react/src/components/DataTable/TableHeader.tsx

```diff
@@ -250,7 +250,7 @@ TableHeader.propTypes = {
    * attribute at the following URL:
    * https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th#attr-scope
    */
-  scope: PropTypes.string.isRequired,
+  scope: PropTypes.string,
```

Comment:
Removing `isRequired`, as we initialize this to `col` but will throw a warning if it is not explicitly provided and `col` is a reasonable default.